deploy_hpo.sh

```diff
@@ -20,31 +20,43 @@ SCRIPTS_DIR="${ROOT_DIR}/scripts"
 HPO_REPO="kruize/hpo"
 HPO_VERSION=$(grep -a -m 1 "HPO_VERSION" ${ROOT_DIR}/version.py | cut -d= -f2)
 HPO_VERSION=$(sed -e 's/^"//' -e 's/"$//' <<<"$HPO_VERSION")
-echo
-echo "Using version: ${HPO_VERSION}"
-HPO_CONTAINER_IMAGE=${HPO_REPO}:${HPO_VERSION}
+
+HPO_SA_MANIFEST="manifests/hpo-operator-sa.yaml"
+HPO_DEPLOY_MANIFEST_TEMPLATE="manifests/hpo-operator-deployment.yaml_template"
+HPO_DEPLOY_MANIFEST="manifests/hpo-operator-deployment.yaml"
+HPO_RB_MANIFEST_TEMPLATE="manifests/hpo-operator-rolebinding.yaml_template"
+HPO_RB_MANIFEST="manifests/hpo-operator-rolebinding.yaml"
+HPO_SA_NAME="hpo-sa"
+HPO_CONFIGMAPS="manifests/configmaps"
 
 #default values
 setup=1
 cluster_type="native"
 CONTAINER_RUNTIME="docker"
+non_interactive=0
+hpo_ns=""
+# docker: loop timeout is turned off by default
+timeout=-1
 
 # source the helpers script
 . ${SCRIPTS_DIR}/cluster-helpers.sh
+. ${SCRIPTS_DIR}/openshift-helpers.sh
 
 function usage() {
-	echo
-	echo "Usage: $0 [-c [docker|minikube|native]] [-o hpo container image]"
-	echo "       -s = start(default), -t = terminate"
-	echo " -c: cluster type."
-	echo " -o: build with specific hpo container image name [Default - kruize/hpo:<version>]"
-	exit -1
+  echo
+  echo "Usage: $0 [-a] [-c [docker|minikube|native|openshift]] [-o hpo container image] [-n namespace] [-d configmaps-dir ]"
+  echo "       -s = start(default), -t = terminate"
+  echo " -c: cluster type."
+  echo " -o: build with specific hpo container image name [Default - kruize/hpo:<version>]"
+  echo " -n: Namespace to which hpo is deployed [Default - monitoring namespace for cluster type minikube]"
+  echo " -d: Config maps directory [Default - manifests/configmaps]"
+  exit -1
```

Comment:
Can you please make sure that the tab formatting is maintained, I can see a number of places in the PR where spaces are being used instead of tabs. For all shell scripts we are using tabs instead of spaces

Reply:
Done